tests/accelerators/test_dp.py

```diff
@@ -134,8 +134,24 @@ def test_step(self, batch, batch_idx):
 
     def training_epoch_end(self, outputs):
         assert outputs[0]["loss"].shape == torch.Size([])
-        assert outputs[0]["reduce_int"].item() == 0  # mean([0, 1]) = 0
-        assert outputs[0]["reduce_float"].item() == 0.5  # mean([0., 1.]) = 0.5
```

Comment:
Shouldn't these two lines stay for `training_epoch_end`? You are not testing anymore the `.item()` equalities

Reply:
they are no longer 0-dimensional tensor since during training we don't reduce anything except the loss. Ther outputs will stay as it is and assertions for these are present in `_assert_extra_outputs` down below.